UtilityApps/src/materialBudget.cpp

```diff
@@ -233,7 +247,7 @@ int main_wrapper(int argc, char** argv)   {
       }
 
 
-      double binX = ( etaMax > 0. ? (0.5+i)*dEta : -theta/M_PI*180. ) ;
+      double binX = ( etaMax > 0. ? (etaMin + (0.5+i)*dEta) : -(thetaMin + theta)/M_PI*180. ) ;
```

Comment:
The `theta` calculation from line 255 already takes `thetaMin` into account, doesn't it?
So that should be the same as before, or not?
```suggestion
      double binX = ( etaMax > 0. ? (etaMin + (0.5+i)*dEta) : -theta/M_PI*180. ) ;
```

Reply:
Ah, yes, you're right.